pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/submission/MessageMetadata.kt

```diff
@@ -0,0 +1,21 @@
+package gov.cdc.ocio.processingstatusapi.models.submission
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+
+/**
+ * MessageMetadata within a report.
+ * @property provenance  Provenance
+
+ */
+@GraphQLDescription("Report metadata containing the disposition type, message identifier, index, aggregation of whether Single or Batch and the filename")
+data class MessageMetadata(
```

Comment:
MessageMetadata should be this:
<img width="725" alt="Screenshot 2024-08-01 at 9 48 33 AM" src="https://github.com/user-attachments/assets/d99d7e3d-016c-4196-989c-56c04ffaef70">